manual_user_install.sh

```diff
@@ -0,0 +1,238 @@
+#!/bin/bash
+
+# tested with a clean 64 bit installation of Raspbian-Lite
+
+function install_core (){
+    echo "Installing OVOS core"
+    echo
+    pip3 install -U pip
+
+    # install ovos-core - ovos-backend-client has conflicting dependencies
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-backend-client
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-core
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-audio
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-ocp-audio-plugin
+    pip3 install tornado
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-messagebus
+    pip3 install adapt-parser
+    pip3 install padacioso
+
+    # dinkum listener
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-dinkum-listener
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-vad-plugin-webrtcvad
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-vad-plugin-silero
+
+    #Precise-lite wake-word cluster
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-ww-plugin-precise-lite
+    python3 -m pip3 install --upgrade setuptools
+    # if pip3 install tflite_runtime is problematic try:
+    # we install full tensorflow (because it just works, takes a while to install)
+    # sudo apt install -y libatlas-base-dev
+    #  pip3 install tensorflow --no-cache-dir
+    pip3 install tflite_runtime
+    pip3 install PyYAML
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-workshop
+    sudo apt install -y python3-fann2
+    pip3 install padatious
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-lingua-franca
```

Comment:
with using git in some packages and other not?

Reply:
Some packages install better with apt, but is fann-2 even needed any more?

Reply:
I will test if fann-2 is needed after we remove other modules.

Reply:
> Some packages install better with apt, but is fann-2 even needed any more?

we migrated to padacioso instead of padatious so it would not, but since padatious can be reenabled in config i'd say leave it for now

Reply:
> Some packages install better with apt, but is fann-2 even needed any more?

what does apt have to do with `pip install git+`?

Reply:
Padatious pulls in its own copy of fann2.  OK if I leave out both padatious and fann2?

Reply:
padatious is optional and disabled by default, users need to enable it in config for latest ovos-core

i think for now leave it out, then in a future PR we can make the script interactive and ask about optional components?

Reply:
After dreaming about this, I opened an issue to discuss a configurator enhancement.  Maybe a bash script isn't the final answer. Ant or make might be more appropriate. The technical key here seems to be efficiently encoding the various dependencies and setup requirements, and the quality metric seems to be how well we can make those rules visible to users.